moped-editor/src/views/projects/projectView/ProjectComponents/ComponentForm.js

```diff
@@ -140,6 +154,14 @@ const ComponentForm = ({
     setValue,
   });
 
+  useResetDependentFieldOnParentFieldChange({
```

Comment:
This clears completion date if the phase is changed so we don't leave a completion date hanging around in the form if there shouldn't be one.